### DISCUSSION TITLE
Time out in 2/3 minutes

### DISCUSSION THREAD

**Discussion by Ipsi-SG** [pallupz/covid-vaccine-booking#197] [Q&A]
My code is getting time out in less than 2/3 minutes and retry is is not happening at a single shot. Giving me the below error:-
Unable to Generate OTP
403 <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML><HEAD><META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=iso-8859-1">
<TITLE>ERROR: The request could not be satisfied</TITLE>
</HEAD><BODY>
<H1>403 ERROR</H1>
<H2>The request could not be satisfied.</H2>
<HR noshade size="1px">
Request blocked.
We can't connect to the server for this app or website at this time. There might be too much traffic or a configuration error. Try again later, or contact the app or website owner.
<BR clear="all">
If you provide content to customers through CloudFront, you can find steps to troubleshoot and help prevent this error by reviewing the CloudFront documentation.
<BR clear="all">
<HR noshade size="1px">
<PRE>
Generated by cloudfront (CloudFront)
Request ID: ajOCEZ6ARCPzu6bYh2uT_e2sZGQ5dBTFQmE8pzhkCnyBGt8JnlSyKQ==
</PRE>
<ADDRESS>
</ADDRESS>
</BODY></HTML>

Is it only with my configuration or someone else facing the same issue.
Any solution for this?

**Comment by pallupz** [comment]
Please try the latest code. Also, are you using a VPN?

**Reply by Ipsi-SG** [reply]
Still not working. Timeout very soon and not getting connected again :(

**Reply by Ipsi-SG** [reply]
I am not using VPN

**Reply by ghost** [reply]
Even with the latest code, the issue is being faced continuously. Is there any solution for this. And I can tell definitely that this was not the case few days back.

**Reply by pallupz** [reply]
Do you have multiple sessions running?

**Reply by pallupz** [reply]
May be more than one scripts or may be in browser and script etc

**Reply by ghost** [reply]
No, not multiple instances and I make sure that I only login via the script and not through browser at the same time. 
My scenario today was 3 comma separated pin codes with refresh frequency of 5 secs.
However the same combination worked fine last week with logout happening at around 10 mins.

**Reply by pallupz** [reply]
I just ran it for half an hour and session timed out just twice - in 15 min intervals as expected

**Reply by ghost** [reply]
That's strange. Not sure then what could be causing this.

**Reply by Ipsi-SG** [reply]
yes till last Saturday it was working perfectly. please help on this.

**Reply by riya2305** [reply]
Validating OTP..
Unable to Validate OTP
Response: {"message": "Endpoint request timed out"}
Retry with ---------- ? (y/n Default y): 

why this error please help

**Reply by pallupz** [reply]
Please use the latest version

**Comment by riya2305** [comment]
facing same issue
